CHANGELOG.md

```diff
@@ -1,13 +1,26 @@
 # Changelog
 
-## [Unreleased]
+## [0.17.0]
 
 ### Added
 
 - Added an experimental swagger definition that includes the specification for
   the vsock API call.
 - Added a signal handler for `SIGBUS` and `SIGSEGV` that immediately terminates
   the process upon intercepting the signal.
+- Added documentation for signal handling utilities.
+- Added [alpha] aarch64 support.
+- Added metrics for read and written bytes by the MMDS, Net and Block Devices.
```

Comment:
nit: it sounds like MMDS is also a device. We can keep the message from the commit message:
```
Add metrics for successful read and write operations of MMDS, Net and Block devices.

Reply:
Ok, done.